server/project/manager.ts

```diff
@@ -505,6 +573,43 @@ const addRoute = async (project: string, routeSrc: string) => {
   return { message: `Route added to project ${project}` };
 };
 
+export const addOperation = async (project: string, operation: string) => {
```

Comment:
I think it might be a good idea to completely regenerate these operations every time routes change. Then it makes it easier to manage when adding and also removing routes. It's server-side operation and parsing is not that costly, so I think we shouldn't worry about the efficiency there.

Reply:
I think a function called `updateTestingClient` that parses the routes and regenerates the `util.ts` file in the testing client would be the best, and you'd plug that into `addRoute` and `deleteRoute` and other relevant places (I don't think there's any other yet)